handlers.c

```diff
@@ -745,17 +745,25 @@ bool handler__lregions(globals_t * vars, char **argv, unsigned argc)
     return true;
 }
 
+static bool do_search ( scan_match_type_t m, globals_t * vars, uservalue_t * pval, char ** argv, unsigned argc );
```

Comment:
Linux kernel coding style with the only exception not to use tabs but 4 spaces for indentation instead.
See: https://www.kernel.org/doc/Documentation/CodingStyle